commit 12f1e5c78213c3f29d498543a26a09e23688c3a0
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2019-05-30T16:04:05+00:00 (4 months ago)
    
    OS-7814 Running top-level "make check" should also check src/fw
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/src/Makefile b/src/Makefile
index 9c7ff263..bb6031c6 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -475,6 +475,7 @@ check: $(JSLINT) $(SUBDIRS)
 		$(JSSTYLE) $(JSSTYLE_OLDSKOOL_OPTS) $$file; \
 		[[ $$? == "0" ]] || exit 1; \
 	done)
+	@$(MAKE) -C fw check
 	@printf "\nJS style ok!\n"
 
 .PHONY: manifest
diff --git a/src/fw/Makefile b/src/fw/Makefile
index 025c264c..fc4b5972 100644
--- a/src/fw/Makefile
+++ b/src/fw/Makefile
@@ -85,7 +85,7 @@ jsstyle:
 	@printf "\n==> Running jsstyle...\n"
 	@# jsstyle doesn't echo as it goes so we add an echo to each line below
 	@(for file in $(JS_CHECK_TARGETS); do \
-		echo $(PWD)/$$file; \
+		echo $(TOP)/$$file; \
 		$(JSSTYLE) $(JSSTYLE_OPTS) $$file; \
 		[[ $$? == "0" ]] || exit 1; \
 	done)
